diff --git a/Makefile b/Makefile
index e769400..4b61927 100644
--- a/Makefile
+++ b/Makefile
@@ -3,7 +3,11 @@ Project=AvailabilityVersions
 include $(MAKEFILEPATH)/CoreOS/Standard/Commands.make
 include $(MAKEFILEPATH)/CoreOS/Standard/Variables.make
 
-VerifierDest	=	/usr/local/libexec
+ifeq ($(SDKROOT),)
+ $(warning SDKROOT not set, xnu build may fail)
+endif
+
+VerifierDest	=	$(SDKROOT)/usr/local/libexec
 
 install:
 	$(_v) $(MKDIR) "$(DSTROOT)$(VerifierDest)"
